### PR TITLE
improve clarity of wiki articles, including label descriptions

### PR DESCRIPTION
**Pull request by indiamos**
Fixed some typos, tidied up syntax and colloquialisms for readers whose first language might not be English, tried to make language and presentation more clear and consistent overall.

This doesn’t address any _officially_ submitted issues that I could find, but I write and proofread a lot of technical documentation in real life, so I figure this is one of the best ways I can contribute. I hope you don’t find it obnoxious! 😽

Sorry if my commits are not squashed. I’m not yet ready to face the horror that is command-line git.

### CODE REVIEW

**Comment by abhisekp** [comment]
@bugron  @ahstro @benschenker please verify this update to the labels descriptions.

File: Wiki-Style-Guide.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
-# What Does "Wiki" Means?
+# What Does “Wiki” Mean?
```
Comment (abhisekp) [inline Wiki-Style-Guide.md]: I don't think we should change the quotes. They were standard quotes.

Symbol|Unicode character|Oct|Dec|Hex|HTML|ref|remark
:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:
“|left double quotation mark|020034|8220|0x201C|`&ldquo;`|http://unicodelookup.com/#“/1|non-standard
"|quotation mark|042|34|0x22|`&quot;`|http://unicodelookup.com/#"/1|standard in coding

File: Wiki-Style-Guide.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 Keep your wiki articles short and focused.
 
 You can include images if they help you better communicate your subject.
 
-## Use descriptive, punctuation-free titles that follow a name convention as `Name of Page`.
+## Use descriptive, punctuation-free titles that follow the naming convention `Name of Page`.
 
-Our wiki articles are [written in GitHub-flavor Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
+Our wiki articles are written in [GitHub-flavored Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
```
Comment (abhisekp) [inline Wiki-Style-Guide.md]: Not concerned with this PR but Markdown-here flavor is different than Github flavored markdown.

File: Wiki.md [hunk 3]
```diff
@@ -21,38 +21,41 @@ Here you will find all wiki related information with regards to guidelines for c
...
+- [More on How to Make a Pull Request from a Fork](Pull-Request-Contribute)
+- [How to Squash Commits](git-rebase#squashing-multiple-commits-into-one): We require only one commit per pull request, unless otherwise specified by a moderator.
 
 ### Labels
 
-Before creating your first pull request or starting your work as an issue moderator it is strongly suggested to get familiar with the [labels guide](Wiki-Labels-Guide). This will provide you with a through understanding on our labeling system so you can proper utilize it and keep a stable workflow.
+Before creating your first pull request or starting your work as an issue moderator, it is strongly suggested that you get familiar with the [labels guide](Wiki-Labels-Guide). This will provide you with a through understanding on our labeling system so you can use it properly and maintain a stable workflow.
 
-For labels on a global levels on FreeCodeCamp's repositories you can follow [this](Select-Issues-for-Contributing-Using-Labels) guide.
+For a list of labels used globally throughout FreeCodeCamp’s repositories, consult the guide “[Select Issues for Contributing Using Labels](Select-Issues-for-Contributing-Using-Labels).”
```
Comment (abhisekp) [inline Wiki.md]: Use standard single and double quotes which are available in usual keyboards. 

Certainly, they don't look as good as the other non-standard pretty quote (which are used widely in written publications, online or offline) but just for consistency across the wiki and for future wiki contributions, we should keep the standard quotes.

File: Wiki-Style-Guide.md (hunk 1 above)
Comment (theflametrooper) [inline Wiki-Style-Guide.md]: I agree with @abhisekp

File: Select-Issues-for-Contributing-Using-Labels.md [hunk 4]
```diff
@@ -1,42 +1,42 @@
...
+In this article you can find labels (see the table below) used to categorize the variety of issues submitted to FCC, along with descriptions (work in progress). The labels range from **bugs** to **questions** to **curriculum requests**. For example, if an issue has a ***Wiki*** label, you can create a wiki article about that issue—after reading the [Wiki Style Guide](Wiki-Style-Guide). 
 
-You can use labels to help you choose which issues to click on, rather than randomly clicking to find one of interest to you. Of course, you can work on and submit PRs for issues without labels. How ever you choose an issue, please be sure to read [Guidelines for Contributing](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) before submitting any pull requests.
+You can use labels to help you choose which issues to click on, rather than randomly clicking to find one of interest to you. Of course, you can work on and submit PRs for issues without labels. However you choose an issue, please be sure to read the [Guidelines for Contributing](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) before submitting any pull requests.
```
Comment (theflametrooper) [inline Select-Issues-for-Contributing-Using-Labels.md]: ~~However you~~ However when you

File: Select-Issues-for-Contributing-Using-Labels.md [hunk 5]
```diff
@@ -1,42 +1,42 @@
...
-| device specific    | The issue presents only for a particular device (smartphone, tablet, desktop, etc).|
-| Discussing         | The issue is currently on discussion stage. |
-| duplicate          | The issue is a duplicate of another issue and most likely will be closed.|
-| easy               | This is used to flag issues that aren't critical, but should be easy for a camper to fix for their first couple of PRs. Think of these as the camper gateway to becoming a contributor. |
+| accessibility      | The issue doesn’t necessarily break the site, but it makes it harder to use (for example, tests used to evaluate while you typed, which would cause the whole screen to flicker). This label also marks improvements to the site’s usability for people with vision impairment or other disabilities.|
+| accounts           | A camper has a problem with his/her FCC account, such as login problems, losing progress, or a profile page that is missing some solutions.|
+| beta               | The issue is related to FCC’s beta site.|
```
Comment (theflametrooper) [inline Select-Issues-for-Contributing-Using-Labels.md]: `’` should be `'`

File: Wiki-Style-Guide.md [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
 
 - [Algorithm Style Guide](Algorithm-Style-Guide)
 
-If you have questions about contributing to the Free Code Camp Wiki message [@rafase282](https://gitter.im/rafase282) in [Gitter](https://github.com/FreeCodeCamp/freecodecamp/wiki/Gitter)
+If you have questions about contributing to the Free Code Camp Wiki, message [@rafase282](https://gitter.im/rafase282) in [Gitter](https://github.com/FreeCodeCamp/freecodecamp/wiki/Gitter).
```
Comment (theflametrooper) [inline Wiki-Style-Guide.md]: ~~message `[@rafase282](https://gitter.im/rafase282)` in~~ message the `[wiki mods](https://gitter.im/FreeCodeCamp/Wiki)` on

**Comment by theflametrooper** [comment]
## LGTM :+1:

Just fix those suggestions :+1:

File: Wiki-Style-Guide.md (hunk 6 above)
Comment (abhisekp) [inline Wiki-Style-Guide.md]: Although not concerned with this PR, 

```diff
- If you have questions about contributing to the Free Code Camp Wiki, message [@rafase282](https://gitter.im/rafase282) in [Gitter](https://github.com/FreeCodeCamp/freecodecamp/wiki/Gitter)
+ If you have any questions regarding contributing to the Free Code Camp Wiki, feel free to contact Wiki Moderators in  <kbd><a href="https://gitter.im/FreeCodeCamp/Wiki"><img src="http://i.imgur.com/ThSWa6Y.png?2"> <b>FreeCodeCamp/Wiki</b></a></kbd>
```
Which will appear as ...

If you have any questions regarding contributing to the Free Code Camp Wiki, feel free to contact Wiki Moderators in  <kbd><a href="https://gitter.im/FreeCodeCamp/Wiki"> <b>FreeCodeCamp/Wiki</b></a></kbd>

**Comment by abhisekp** [comment]
Thanks for this one. :tada: :smiley: 

Let it keep coming :heartpulse:
<br>
<br>

> ----
<p align="center">Need Help? Read <a href="https://github.com/FreeCodeCamp/wiki/blob/master/.github/CONTRIBUTING.md"><b>CONTRIBUTING</b></a> Guidelines<br>or Chat with us in <kbd><a href="https://gitter.im/FreeCodeCamp/Wiki"> <b>FreeCodeCamp/Wiki</b></a></kbd></p>

> ----

**Comment by abhisekp** [comment]
@indiamos also you might enjoy this one :point_right: [**Wiki Labels**](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Wiki-Labels-Guide) :dancers::dancers::dancers:

File: Wiki-Style-Guide.md (hunk 6 above)
Comment (theflametrooper) [inline Wiki-Style-Guide.md]: Yeh @abhisekp solution is the one to go with!

File: Select-Issues-for-Contributing-Using-Labels.md (hunk 4 above)
Comment (indiamos) [inline Select-Issues-for-Contributing-Using-Labels.md]: _Well, actually…_

“However” in the original text was being used in its secondary sense of “in whatever way”;* it was just misspelled. More wordily, it was saying, “Whether you choose an issue by using labels or not, RTFM.”

If you add a comma, however, that “However” switches to its most common meaning: “to introduce a statement that contrasts with or seems to contradict something that has been said previously.”* There’s no contradiction with what precedes it, though; “please be sure to read the Guidelines” applies to all issues, regardless of how they are chosen.

\* New Oxford American Dictionary

Comment (alayek) [inline Select-Issues-for-Contributing-Using-Labels.md]: Gram-trooper (I was going for Grammer Nazi, but oh well!) @theflametrooper, meet your match!

Comment (indiamos) [inline Select-Issues-for-Contributing-Using-Labels.md]: 😻 That is the best username, @theflametrooper! 😻

**Comment by Rafase282** [comment]
@indiamos let me know when you make the changes.

**Comment by Rafase282** [comment]
Would you mind changing `easy` to `first timers only` as per https://github.com/FreeCodeCamp/wiki/issues/866

**Comment by theflametrooper** [comment]
@indiamos Due to wanting to get this PR fixed before the May Summit, I'm gonna merge your work and submit a PR of my own to fix the bugs. Thanks for your contribution!

File: Select-Issues-for-Contributing-Using-Labels.md (hunk 4 above)
Comment (theflametrooper) [inline Select-Issues-for-Contributing-Using-Labels.md]: I see now, thanks ;)

File: Select-Issues-for-Contributing-Using-Labels.md [hunk 7]
```diff
@@ -1,42 +1,42 @@
...
 
-You can use labels to help you choose which issues to click on, rather than randomly clicking to find one of interest to you. Of course, you can work on and submit PRs for issues without labels. How ever you choose an issue, please be sure to read [Guidelines for Contributing](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) before submitting any pull requests.
+You can use labels to help you choose which issues to click on, rather than randomly clicking to find one of interest to you. Of course, you can work on and submit PRs for issues without labels. However you choose an issue, please be sure to read the [Guidelines for Contributing](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) before submitting any pull requests.
 
-Check out the table below for the list of FreeCodeCamp issue labels, along with descriptions (work in progress). Thanks @bugron, @ahstro, @benschenker, for compiling this list of labels and descriptions.
+Thanks @bugron, @ahstro, @benschenker, for compiling this list of labels and descriptions.
```
Comment (abhisekp) [inline Select-Issues-for-Contributing-Using-Labels.md]: Should this be kept as is or removed?

Comment (theflametrooper) [inline Select-Issues-for-Contributing-Using-Labels.md]: I would remove it. Just saying...

**Pull request merged**